src/app/AppLayout/AppLayout.tsx

```diff
@@ -271,7 +326,25 @@ const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
   const Header = (
     <>
       <PageHeader
-        logo={<Brand alt="Cryostat" src={cryostatLogo} className="cryostat-logo" />}
+        logo={
+          <>
```

Comment:
Also, are there any color guide on the labels? Like Dev should be red and Beta should be green?

Reply:
I tried the same `Level hasGutter` trick but the label no longer being aligned with the logo made it look worse, IMO.

I don't know that there is any colour guidance for this kind of thing, since feature flags aren't a PF component or concept.

Reply:
Ahh thats unfortunate. How about adding `&nbsp;&nbsp;`?

```diff
diff --git a/src/app/AppLayout/AppLayout.tsx b/src/app/AppLayout/AppLayout.tsx
index 8d6a635..800e5a9 100644
--- a/src/app/AppLayout/AppLayout.tsx
+++ b/src/app/AppLayout/AppLayout.tsx
@@ -329,6 +329,7 @@ const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
         logo={
           <>
             <Brand alt="Cryostat" src={cryostatLogo} className="cryostat-logo" />
+            &nbsp;&nbsp;
             <FeatureFlag strict level={FeatureLevel.DEVELOPMENT}>
               <PageHeaderToolsItem>
                 <Label isCompact color="red">
```

Reply:
![Screenshot from 2022-12-19 13-45-01](https://user-images.githubusercontent.com/68053619/208497420-b5c11d49-01d9-4dcd-92d3-7a1a3c50e28a.png)

Reply:
I guess... or maybe I'll just add an inline style to the label to give it some extra margin or something.

Reply:
> I don't know that there is any colour guidance for this kind of thing, since feature flags aren't a PF component or concept.

Oh okayy then I think the current colors are good already.

Reply:
> I guess... or maybe I'll just add an inline style to the label to give it some extra margin or something.

I think thats even better :D